test/Directory.Build.props

```diff
@@ -21,4 +21,10 @@
   <ItemGroup>
     <Compile Include="$(MSBuildThisFileDirectory)GlobalSuppressions.cs" Link="GlobalSuppressions.test.cs" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <!-- reference assemblies let us target .NET Framework without the SDK (for example, on non-Windows) -->
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="all" />
```

Comment:
Is there a good way to add this _everywhere_?

Reply:
I really just added on projects that were failing to built, that was my intent at least. I don't know if we want this as a global thing.

Reply:
I was just wondering if we should try. Definitely not blocking your PR.

We already it include it in some top-level `props` files, but not all. Was wondering if we should make it more global since it doesn't hurt to have it even in projects that don't need it.

Reply:
Yeah, makes sense, I will try that in another PR.